packages/cli/bin.js

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env node --experimental-specifier-resolution=node --no-warnings
+"use strict";
+// eslint-disable-next-line import/no-internal-modules
+import "./lib/bin";
```

Comment:
Don't forget extension

Reply:
esbuild doesn't generate extensions on imports, so do I

Reply:
See ./lib folder

Reply:
Please add it. I'll figure out what to do with generated modules.